Source/Scene/BillboardCollection.js

```diff
@@ -1482,11 +1483,27 @@ define([
                     vs.defines.push('DISTANCE_DISPLAY_CONDITION');
                 }
 
+                fs = new ShaderSource({
```

Comment:
Here and in the point cloud, shouldn't the render state be different for each pass?  Opaque writes depth; translucent does not.